d2b_migrate.module

```diff
@@ -701,7 +721,7 @@ function d2b_migrate_backup_form_backup($form, &$form_state) {
   $database = config_get('d2b_migrate.settings', 'source_db_machine_name');
   $settings->source_id = $database;
 
-  backup_migrate_ui_manual_backup_perform($settings);
+  backup_migrate_perform_backup($settings);
```

Comment:
Unrelated change but this has no impact. `backup_migrate_ui_manual_backup_perform()` literally just wraps around `backup_migrate_perform_backup()`.